src/components/PDFViewer/PDFViewer.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import PDFViewer from 'pdf-viewer-reactjs'
+import { useStage } from '../../context/StageContext'
+
+const ExamplePDFViewer = () => {
+    const {pdfUrl} = useStage()
+    return (
+        <PDFViewer
+            document={{
```

Comment:
This seems to work fine, we should try to push it and try to break it, just in case the pdfUrl may come as null and and error can arise